src/components/Dashboard/FirmsAddressList.vue

```diff
@@ -133,7 +133,11 @@ export default class FirmsAddressList extends Mixins(CommonMixin, CountriesProvi
   @Getter getBusinessAddress!: OfficeAddressIF
 
   // Business
-  get businessAddress (): OfficeAddressIF {
+  get businessAddress (): OfficeAddressIF|null {
+    let deliveryAddress = this.getBusinessAddress?.deliveryAddress
+    if (this.getBusinessAddress === null) return null
+    if (deliveryAddress === null) return null
+    if (Object.values(deliveryAddress).length === 0) return null
```

Comment:
Again, please use isEmpty() instead of object values.

Rationale: Object.values() is looking deeply into the object's implementation, which we don't need/want to do at our "high level Typescript code" level.

Reply:
Thanks. I've changed it to isEmpty(). Simple, elegant and powerful.